src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/AsyncTest/BeginExecAsyncTest.cs

```diff
@@ -36,36 +36,16 @@ public static void ExecuteTest()
         {
             using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
             {
-                try
-                {
-                    SqlCommand command = new SqlCommand(GenerateCommandText(), connection);
-                    connection.Open();
-
-                    IAsyncResult result = command.BeginExecuteNonQuery();
-                    while (!result.IsCompleted)
-                    {
-                        System.Threading.Thread.Sleep(100);
-                    }
+                SqlCommand command = new SqlCommand(GenerateCommandText(), connection);
```

Comment:
I think you modified this test in this PR already; https://github.com/dotnet/SqlClient/pull/1155/files#diff-0cae55c504ba126a6ef31c093528ee07c311aaac14d99b8e9ad0374141758d40R39

Changes can be reversed to avoid conflicts.